.github/workflows/verify-pr-description.yml

```diff
@@ -0,0 +1,37 @@
+name: Verify Pull-Request Description
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+permissions: read-all
+
+jobs:
+  verify:
+    if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
info: I've added this check, so PR template won't be verified for dependabot PRs